src/utils/currencyId.ts

```diff
@@ -1,5 +1,12 @@
 import { Currency } from '@uniswap/sdk-core'
 
+// swap router API special cases these strings to represent native currencies
+// all chains have "ETH" as native currency symbol except for polygon
+export enum SwapRouterNativeAssets {
```

Comment:
this feels like it will break something in the future.

i think a similar thing caused the price charts to break recently. hopefully we are standardizing the DB keys for native assets

Reply:
so these are constants supported by the routing-api, not token keys stored in any db. we'll have to pass in either a string `ETH` or just make an enum

Reply:
can they be imported from the routing package?

Reply:
i think longer-ish term i'd prefer to have routing-api support `NATIVE` as a string and then have it be exported from sdk-core. That way it would match up with our string addresses for our graphql and amplitude endpoints.